### PR TITLE
Always trim ending slashes from URL in apiclient

### LINKED ISSUE #172: V2.0.x Connection Failure
**Describe the bug**
Hi, I have not been able to login to the app after the latest v2.0.0 auto updated from the Play Store. It has been working fine with the cordova version till this upgrade. Now both the local LAN IP and reverse proxy is not working on the v2.0.0. I've confirmed that the WebUI, Tizen, Android TV and iPhone client still works. For WebUI and Android TV I tested on the same device as the v2.0.0. I've tried reinstalling and clearing the cache/data no luck. Previously I remember signing up for the beta and it works but I opted out then reinstalled the cordova version due to subtitles issue in the beta. Think it's been only less than two days v2.0.0 released in the Play Store and I noticed there's no such reported issue so I appreciate if someone could give me some advise on this. Thank you.

**To Reproduce**

1. Launch the app.
2. Fill up the server details or from choose server.
3. Log in with a working account.
4. Kept seeing this "Connection Failure" error screenshot attached.

**Expected behavior**
App works normally after login.

**Logs**
No logged activity from Jellyfin Server.

**Screenshots**
Screenshot_20201007-234509_Jellyfin

**System (please complete the following information):**
 - App version: 2.0.0
 - Android version: Android 10
 - Device manufacturer and model: Samsung S9+
 - Jellyfin server version: 10.6.4

### PR DESCRIPTION
**Pull request by nielsvanvelzen**
This should (partially?) fix #172

### CODE REVIEW

**Review by Maxr1998 — APPROVED** [review]
Looks good, waiting for user feedback before merging however.

**Comment by nielsvanvelzen** [comment]
@Maxr1998 since this fixes at least _some_ issues I'd say we can merge it now

**Comment by Maxr1998** [comment]
Wanna do a rebase before that? Keep the history a bit cleaner.

**Pull request merged**
